From 5e91829b501d30214ecd487004b2907b26f26992 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Sat, 3 Jan 2026 14:29:47 +0800
Subject: [PATCH 56/63] loonggpu-bridge: make some functions static

They are only used in the TU where the function is defined.  Thus we
can make them static to fix some missing prototype warnings.

Signed-off-by: Xi Ruoyao <xry111@xry111.site>
---
 loonggpu-bridge/it66121_drv.c | 2 +-
 loonggpu/loonggpu_dc_dp.c     | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/loonggpu-bridge/it66121_drv.c b/loonggpu-bridge/it66121_drv.c
index 8062dd2..22d2e34 100644
--- a/loonggpu-bridge/it66121_drv.c
+++ b/loonggpu-bridge/it66121_drv.c
@@ -26,7 +26,7 @@ DEFINE_SIMPLE_REG_SEQ_FUNC(afe_high)
 DEFINE_SIMPLE_REG_SEQ_FUNC(afe_low)
 DEFINE_SIMPLE_REG_SEQ_FUNC(hdmi)
 
-int it66121_hdmi_output_mode(struct loonggpu_bridge_phy *phy, int mode)
+static int it66121_hdmi_output_mode(struct loonggpu_bridge_phy *phy, int mode)
 {
 	if ((enum hdmi_mode)mode == HDMI_MODE_NORMAL) {
 		SIMPLE_REG_SEQ_FUNC(hdmi)(phy);
diff --git a/loonggpu/loonggpu_dc_dp.c b/loonggpu/loonggpu_dc_dp.c
index aff48e0..aea2cb7 100644
--- a/loonggpu/loonggpu_dc_dp.c
+++ b/loonggpu/loonggpu_dc_dp.c
@@ -852,7 +852,7 @@ int ls2k3000_dp_resume(struct loonggpu_dc_crtc *crtc, int intf)
 	return 0;
 }
 
-int ls2k3000_dp_aux_detect_status(struct loonggpu_dc_crtc *crtc, int intf)
+static int ls2k3000_dp_aux_detect_status(struct loonggpu_dc_crtc *crtc, int intf)
 {
 	struct loonggpu_device *adev = crtc->dc->adev;
 	struct connector_resource *connector_res;
-- 
2.52.0

